runtime/proto/service/layout.proto

```diff
@@ -46,13 +40,7 @@ message ProposeLayoutRequestMsg {
 }
 
 message ProposeLayoutResponseMsg {
-  enum Type {
-    INVALID = 0;
-    ACK = 1;
-    REJECT = 2;
-  }
-
-  Type resp_type = 1;
+  bool proposed = 1;
```

Comment:
please bring back enums. Boolean variables much harder to understand - the value doesn't explain domain meaning, and that brings additional complexity and makes code more indirect

Reply:
Please address this.